libs/application/core/src/lib/messages.ts

```diff
@@ -0,0 +1,127 @@
+    defaultMessage: 'Úps! Eitthvað fór úrskeiðis',
+    description: 'Oops! Something went wrong',
+  },
+  globalErrorMessage: {
+    id: 'application.system:boundary.error.message',
+    defaultMessage:
+      'Fyrirgefðu! eitthvað fór rosalega úrskeiðis og við erum að skoða það',
+    description:
+      'Sorry! something went terribly wrong and we are looking into it',
+  },
+  userRoleError: {
+    id: 'application.system:user.role.error',
+    defaultMessage:
+      'Innskráður notandi hefur ekki hlutverk í þessu umsóknarástandi',
+    description:
+      'Logged in user does not have a role in this application state',
+  },
+  notFoundTitle: {
+    id: 'application.system:notFound',
+    defaultMessage: 'Umsókn finnst ekki',
+    description: 'Application not found',
+  },
+  notFoundSubTitle: {
+    id: 'application.system:notFound.message',
+    defaultMessage: 'Engin umsókn fannst á þessari slóð.',
+    description: 'No application was found at this URL.',
+  },
+  notFoundApplicationType: {
+    id: 'application.system:notFound.application.type',
+    defaultMessage: 'Þessi gerð umsókna er ekki til',
+    description: 'This type of application does not exist',
+  },
+  notFoundApplicationTypeMessage: {
+    id: 'application.system:notFound.application.message',
+    defaultMessage: 'Engin umsókn er til af gerðinni: {type}',
+    description: 'There is no application of the type: {type}',
+  },
+  createErrorApplication: {
+    id: 'application.system:create.error.application',
+    defaultMessage: 'Eitthvað fór úrskeiðis',
+    description: 'Something went wrong',
+  },
+  createErrorApplicationMessage: {
+    id: 'application.system:create.error.application.message',
+    defaultMessage: 'Ekki tókst að búa til umsókn af gerðinni: {type}',
+    description: 'Failed to create application of type: {type}',
+  },
+  applications: {
+    id: 'application.system:applications',
+    defaultMessage: 'Þínar umsóknir',
+    description: 'Your applications',
+  },
+  newApplication: {
+    id: 'application.system:new.application',
+    defaultMessage: 'Ný umsókn',
+    description: 'New application',
+  },
+  tagsInProgress: {
+    id: 'application.system:tags.inProgress',
+    defaultMessage: 'Í vinnslu',
+    description: 'In progress',
+  },
+  tagsRequiresAction: {
+    id: 'application.system:tags.requiresAction',
+    defaultMessage: 'Krefst aðgerða',
+    description: 'Requires action',
+  },
+  thanks: {
+    id: 'application.system:thanks',
+    defaultMessage: 'Takk fyrir',
+    description: 'Thank you',
+  },
+  thanksDescription: {
+    id: 'application.system:thanks.description',
+    defaultMessage:
+      'Úrvinnslu þinni er lokið. Umsókn er komin áfram í ferlinu.',
+    description:
+      'Your application is complete. The application has progressed in the process.',
+  },
+  notLoggedIn: {
+    id: 'application.system:not.logged.id',
+    defaultMessage: 'Þú þarft að vera skrá þig inn.',
```

Comment:
The word skrá is not correct here. It should be either skráður (male) or skráð (female/other).

Reply:
Not gonna lie all the translations are done by Google translate for the ones I found in English only. Would need someone better than me to review that

Reply:
I'm gonna figure it out to proof read the copy :)